tests/runner/src/tests/smoke.rs

```diff
@@ -3,11 +3,53 @@ use fluvio_command::CommandExt;
 
 use fluvio_integration_derive::fluvio_test;
 use fluvio::Fluvio;
-use fluvio_test_util::test_meta::TestOption;
+use fluvio_test_util::test_meta::TestCase;
 use std::sync::Arc;
 
+// TODO: Validate override keys:
+// use_cli (default false)
+// producer.iteration (default 1)
+// producer.record_size (default 100)
+// consumer.wait (default false)
+
+// Add logic for this in future
+// Disable consumer
+// Disable producer
+// Producer count
+
 #[fluvio_test]
-pub async fn run(client: Arc<Fluvio>, option: TestOption) {
+pub async fn run(client: Arc<Fluvio>, option: TestCase) {
+    // Communicate what our expected smoke test values are.
+    // We will need a more robust pattern for all tests to validate inputs
+    // This is unwieldy duplication. But this is a small step forward
+    println!("\n###\nSmoke test vars: (Override with `-v key=val`)");
```

Comment:
Can we just centralize logging for Option in once place instead of putting into test?

Reply:
I think so, yes. The purpose of this logging was more about over-communicating because I found the lack of input validation for custom vars difficult to use. I don't think we're done iterating with how it will work if we want to add more tests with their own variables, but I didn't have time to make it perfect without feedback from everyone else.